src/cart/crt_rpc.h

```diff
@@ -92,10 +92,12 @@ struct crt_common_hdr {
 	uint32_t	cch_flags;
 	/* HLC timestamp */
 	uint64_t	cch_hlc;
-	/* gid and rank identify the rpc request sender */
-	d_rank_t	cch_rank;
+	/* destination rank in default primary group */
+	d_rank_t	cch_dst_rank;
+	/* originator rank in default primary group */
+	d_rank_t	cch_src_rank;
```

Comment:
For replies, cch_dst_rank (or cch_src_rank) contains the destination (or origination) of the request, not of the reply, correct? Just to make sure I understand these two correclty.

Reply:
For replies src/dst would be from perspective of the caller, so in the callers completion handler cch_src_rank would be itself (the rank that issues the rpc call), and cch_dst_rank/tag will contain remote ep it was trying to reach in this rpc.